src/Validator.php

```diff
@@ -34,8 +35,11 @@ public function __construct(private RuleHandlerResolverInterface $ruleHandlerRes
     public function validate(mixed $data, ?iterable $rules = null): Result
     {
         $data = $this->normalizeDataSet($data);
+
         if ($rules === null && $data instanceof RulesProviderInterface) {
```

Comment:
could you also change validator collector in the debug package?

Reply:
done.